client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -52,9 +52,9 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 				answer={ translate(
 					'Yes! With the WordPress.com Business plan you can search for and install external plugins.' +
 						' All plans already come with a custom set of plugins tailored just for them.' +
-						' {{a}}Check out all included plugins{{/a}}.',
+						' {{a}}Find out more about plugins{{/a}}.',
 					{
-						components: { a: <a href={ `/plugins/${ siteSlug }` } /> },
+						components: { a: <a href="https://en.support.wordpress.com/plugins/" /> },
```

Comment:
I don't think we want to hardcode the `en` English version here; we should be checking the locale of the current user and plugging that in (eg. es.support.wordpress.com`). If the site doesn't exist, eg. `fr.support.wordpress.com`, it'll forward to `en` anyways. Some recent background discussion can be found at pxLjZ-4Cu-p2.

Reply:
Good point.

I did a bit more digging and found a few things.

1. It looks like this is in a pretty bad state across Calypso.  The majority of [instances of `support.wordpress.com`](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=support.wordpress.com&type=) link to the `en` subdomain.
2. There are some links to plain `support.wordpress.com` in there, but I would like to know more about what this does before using it.
3. There is a `getSupportLocale` method which could be extracted out to a utility library and re-used.  It will return one of (`en`, `es`, `pt-br`) which I gather are the locales we think are the best-translated.  BUT:
4. https://es.support.wordpress.com/es-plugins/ has no mention of the Business plan allowing plugin support.  This makes sense given that we don't have a strategy for keeping support page content up to date.
5. The content of https://pt-br.support.wordpress.com/plugins/ is in English anyway.

So overall, I think it might actually best to link to the English version here.  For non-English users, browsers should offer to translate it automatically, which is going to be better than what we can offer at this point.

Another option would be to update the Spanish translation for this change and go with `getSupportLocale` - I would be fine with that too.